src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
```

Comment:
I understand what you mean, at the end is like we are returning a string instead of an Error. I don't think it will cause any issue but I agree that it's better to uniform all the promise's returned values so I'll add it. Regarding the stringified JSON, I think it's better to display its content as it gives to the user more information about the issue. Taking a look at the documentation, it seems that there can be just two errors: `404 - not found` and `500 - Please correct error on config.xml file in the project folder.` If you want I can manage this two cases and deliver a better output.